packages/styles/forms.css

```diff
@@ -182,10 +182,16 @@ textarea.Field--has-error:focus:hover,
 
 .Error {
   color: var(--field-error-text-color);
-  border-top: 1px solid var(--field-error-border-color);
   text-align: left;
-  font-weight: var(--font-weight-normal);
   font-size: var(--text-size-smallest);
```

Comment:
Think this was done in your previous PR as well, per the `git blame`. Was it left in intentionally?

Reply:
I believe the font-weight does need to stay out as we were trying to standardize on that.